### ISSUE TITLE
Color not changing on seletion

### ISSUE THREAD

**Issue by Miteshmakwana73** [aurelhubert/ahbottomnavigation#351]
This is my code, my problem is after applying `setOnTabSelectedListener` it stops working to change item color. if I remove `setOnTabSelectedListener` it's working great

what should I do, please suggest me
`public class MainActivity extends AppCompatActivity {
    private AHBottomNavigation bottomNavigation;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        boolean enabledTranslucentNavigation = getSharedPreferences("shared", Context.MODE_PRIVATE)
                .getBoolean("translucentNavigation", false);
        setTheme(enabledTranslucentNavigation ? R.style.AppTheme_TranslucentNavigation : R.style.AppTheme);
        bottomNavigation = findViewById(R.id.bottom_navigation);
        setupbottomnavigation();

    }

    private void setupbottomnavigation() {
        // Create items
        AHBottomNavigationItem item1 = new AHBottomNavigationItem(R.string.Home, R.drawable.ic_home_black_24dp, R.color.color_tab_1);
        AHBottomNavigationItem item2 = new AHBottomNavigationItem(R.string.Search, R.drawable.ic_search_black_24dp, R.color.color_tab_2);
        AHBottomNavigationItem item3 = new AHBottomNavigationItem(R.string.Category, R.drawable.ic_apps_black_24dp, R.color.color_tab_3);
        AHBottomNavigationItem item4 = new AHBottomNavigationItem(R.string.Favourite, R.drawable.ic_favorite_black_24dp, R.color.color_tab_4);

        // Add items
        bottomNavigation.addItem(item1);
        bottomNavigation.addItem(item2);
        bottomNavigation.addItem(item3);
        bottomNavigation.addItem(item4);

        bottomNavigation.setDefaultBackgroundColor(Color.parseColor("#FEFEFE"));

        // Change colors
//        bottomNavigation.setInactiveColor(Color.parseColor("#000000"));

        bottomNavigation.setForceTint(true);
        bottomNavigation.setColored(false);

//        bottomNavigation.setTranslucentNavigationEnabled(true);

        bottomNavigation.setTitleState(AHBottomNavigation.TitleState.ALWAYS_HIDE);

        bottomNavigation.setOnNavigationPositionListener(new AHBottomNavigation.OnNavigationPositionListener() {
            @Override
            public void onPositionChange(int y) {
                showToast(String.valueOf(y));
            }
        });

        bottomNavigation.setOnTabSelectedListener(new AHBottomNavigation.OnTabSelectedListener() {
            @Override
            public boolean onTabSelected(int position, boolean wasSelected) {
                showToast(String.valueOf(position));
                return false;
            }
        });
//        bottomNavigation.setCurrentItem(1)
    }

    private void showToast(String msg) {
        Toast.makeText(this, msg, Toast.LENGTH_SHORT).show();
    }
}`

Any help would be highly appreciated.

**Comment by dario30186** [comment]
I am having the same issue, did you find a solution ?

**Comment by Miteshmakwana73** [comment]
No, i use [](https://github.com/ittianyu/BottomNavigationViewEx) this library

**Comment by 455361051** [comment]
you should return true in "onTabSelected" .
    bottomNavigation.setOnTabSelectedListener(new AHBottomNavigation.OnTabSelectedListener() {
        @Override
        public boolean onTabSelected(int position, boolean wasSelected) {
            showToast(String.valueOf(position));
            return true;
        }
    });
